src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -516,6 +516,11 @@
     "NAME": "Oak Leaf",
     "DESCRIPTION": "Gain !M! Dexterity."
   },
+  "stsjorbsmod:OldBook": {
+    "NAME": "Old Book",
+    "DESCRIPTION": "stsjorbsmod:Entombed (Die). NL. Unplayable. NL stsjorbsmod:Destroy this.",
```

Comment:
I think the `Entombed (Die)` syntax was just shorthand in the spreadsheet; the parenthetical isn't something we've used in other cards, so I'm not sure this will necessarily be clear to a player. I'd suggest something like:

```stsjorbsmod:Entombed. NL stsjorbsmod:Exhume when you would die; instead, stsjorbsmod:Destroy this card. NL Unplayable.```